src/component/plotly-graph/inject.js

```diff
@@ -38,7 +41,7 @@ function inject (opts = {}) {
   }
 
   if (isNonEmptyString(plotlyJS)) {
-    let src = resolve(plotlyJS)
+    let src = resolvePlotlyJS(plotlyJS)
```

Comment:
I'd vote for adding an CLI / API option here. For example a boolean `--fetch-plotly.js` or more generally `--fetch-ressources` as some users might still want to leave it to the electron to app to fetch their custom (i.e. non-CDN) plotly.js bundles.